vispy/visuals/mesh.py

```diff
@@ -136,6 +131,21 @@
 """)
 
 
+_null_color_transform = 'vec4 pass(vec4 color) { return color; }'
+_clim = 'float cmap(float val) { return (val - $cmin) / ($cmax - $cmin); }'
+
+
+def _build_color_transform(data, cmap, clim=(0., 1.)):
```

Comment:
Is it worth it to try to combine this `_build_color_transform` with the one in `image.py`? I know in one of my own projects that uses vispy I end up using clims with an ImageVisual subclass.

Reply:
It looks like `visuals/image.py` (line ~92) uses a transformation of `(color.r + color.g + color.b / 3.)` whereas this just takes and converts a `float`. So it's not quite so straightforward unfortunately. I'll add a comment that eventually they could be unified, though.